roles/security_errata/tasks/main.yml

```diff
@@ -18,6 +18,7 @@
 
 - include: CVE-2015-0235.yml
   tags: CVE-2015-0235
+  when: ansible_architecture != "ppc64le"
```

Comment:
This could use some more context. Why would you be opting out of a security playbook based on arch?